app/controllers/plan_wizard_controller.rb

```diff
@@ -88,8 +103,12 @@ def perform_step
   def work_plan_params
     return {} unless params[:work_plan]
     params.require(:work_plan).permit(
-      :original_set_uuid, :project_id, :product_id, :product_options, :comment, :desired_date, :work_order_id, :work_order_modules, :work_order_module => {}
+      :original_set_uuid, :project_id, :product_id, :product_options, :comment, :desired_date, :data_release_strategy_id, :work_order_id, :work_order_modules, :work_order_module => {}
     )
   end
 
+  def is_product_from_ss?
+    @work_plan.product.catalogue.url == Rails.configuration.ss_data_release_url
```

Comment:
Please could you change this to config.data_release_url (so it's consistent with the Projects app)